projects/ngx-prx-styleguide/src/lib/select/select.component.spec.ts

```diff
@@ -10,7 +10,7 @@ import { SelectComponent } from './select.component';
       [selected]="testSelected"
       [options]="testOptions"
       [single]="testSingle"
-      (onSelect)="setTestOutput($event)">
+      (select)="setTestOutput($event)">
```

Comment:
was this was also painful? maybe tslint-disable?

Reply:
Wasn't too bad in Publish. Are these changes going to be relevant anywhere else do you think? Metrics has some selects, right?